pkg/ebpf/tracee.go

```diff
@@ -79,6 +79,11 @@ type netProbe struct {
 	egressHook  *bpf.TcHook
 }
 
+// RequiredInitValues determines if to initialize values that might be needed by eBPF programs
+type RequiredInitValues struct {
+	kallsyms bool
+}
+
```

Comment:
In my opinion - it is important.
It is important because it is expandable. If there will be more requirements for initialization, you have a very convenient way to store them without changing the code. Its use will be the greatest if this value is also determined from iterations on the events, which will mean you won't need to change any function interface to add it, just add some code to the `handleDependencies` function.

Reply:
It is possible to transform this value to a list of requested symbols like you requested in other comment.